reconcile/terraform_repo.py

```diff
@@ -361,6 +361,8 @@ def calculate_diff(
                 )
             if self.params.validate_git:
                 self.check_ref(d.repository, d.ref)
+            if d.force_rerun_timestamp:
```

Comment:
how is `force_rerun_timestamp` stored in current state, if not persisted, will this always trigger rerun?

Reply:
The value will be stored in S3 state here: https://github.com/app-sre/qontract-reconcile/blob/828dd8293828dec3e33536358d71620a94e7671b/reconcile/terraform_repo.py#L270 and considered just like any other value in the diffing process so after its written to state once, it should not trigger a subsequent re-run until the value is updated or removed.